tools/webpack-plugin/src/models/AssetStats.ts

```diff
@@ -0,0 +1,9 @@
+import { UploadResult } from '@backtrace/sourcemap-tools';
+
+export interface AssetStats {
```

Comment:
Let's think about what we have here. The purpose of the debugId property is to generate an output for the user in the source map. Why are we mixing it together with errors etc. It doesnt' sound right to me